### DISCUSSION TITLE
Added battery powered camera - no stream, no snapshot - but img ok.

### DISCUSSION THREAD

**Discussion by dwdonnelly** [mrlt8/docker-wyze-bridge#1303] [Q&A] [answered]
@mrlt8 I have added a new external battery-powered camera to the docker wyze bridge.  In the docker wyze bridge camera views, when I click "Streams" and select RSTP Snapshot, a new browser tab opens with "404: Not Found".

Also, when I add a generic camera for my dashboard, I can set the Still Image field with `http://homeassistant.local:5000/img/deck-camera.jpg?api=<my API key>`      Note - img works to get an image, but snapshot does not.  The "img" does update with a new img.  The camera is staring at me, so I am sure it is active.
First question:  Should I worry about the img vs snapshot source for the displayed picture?  The docker wyze camera views and the dashboard generic camera still image do update correctly. 

Also, I can not get the Stream Source to work.  I have tried configuring this new camera as my other outdoor cameras are configured. 
 I used `rtsp://wb:<my API key>@homeassistant.local:8554/deck-camera`   But I got an error "Timeout while loading URL"
I also tried `rtsp://127.0.0.1:8554/deck-camera`   But I get an error: "Timeout while loading URL"

Your help is appreciated.

**Comment by mrlt8** [comment]
Do the streams work directly in the webUI and or in another media player like VLC?

**Comment by dwdonnelly** [comment, marked as answer]
I put this in VLC: `rtsp://wb:<my api key>@homeassistant.local:8554/deck-camera` and yes - it shows correctly.

And yes - I then copied same `rtsp://wb:<my api key>@homeassistant.local:8554/deck-camera` and placed in the generic camera set up.  AND EMBARRASSINGLY - it worked.  Unbelievable! 

I do not know what to say?????  It works.  You must have brought good carma to the issue.  THANK YOU!

This issue simply solved.  THANK YOU !

PS:  Do I understand correctly - img or snapshot is fine to use?
